src/cmd/root.go

```diff
@@ -156,7 +157,15 @@ var rootCmd = &cobra.Command{
 				}),
 			),
 			func(ctx context.Context, req mcp.CallToolRequest) (*mcp.CallToolResult, error) {
-				resp, err := client.ListTeams(nil)
+				searchTerm := req.GetString("searchTerm", "")
+
+				var resp *opslevel.TeamConnection
+				var err error
+				if searchTerm != "" {
+					resp, err = client.SearchTeams(searchTerm, nil)
+				} else {
+					resp, err = client.ListTeams(nil)
+				}
```

Comment:
Would using a "blank" searchTeam just return all teams?  If so this could be simplified to just use `client.SearchTeams`

```suggestion
			    resp, err = client.SearchTeams(searchTerm, nil)
```